src/test/util_tests.cpp

```diff
@@ -754,7 +754,7 @@ BOOST_AUTO_TEST_CASE(test_LockDirectory)
     thr.join();
     BOOST_CHECK_EQUAL(threadresult, true);
 #ifndef WIN32
-    // Try to aquire lock in child process while we're holding it, this should fail.
+    // Try to acquire lock in child process while we're holding it, this should fail.
```

Comment:
Already reported in #12543.

Reply:
So should I revert this one or just do nothing?

Reply:
Revert or rebase on top of master now that #12543 is merged.

Reply:
Reverted. Not familiar with git rebase and such...